.idea/.generators

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Could you remove the RubyMine files (`.idea/*`) from your PR? If it'd be helpful to add `.idea` to `.gitignore`, feel free to do that!